Source/Plugins/Core/com.equella.core/js/tsrc/cloudcontrol/index.ts

```diff
@@ -58,18 +60,20 @@ var registrations: {
   [key: string]: Registration | undefined;
 } = {};
 var listeners: ((doc: ItemState) => void)[] = [];
+var controlValidators: { validator: ControlValidator; ctrlId: string }[] = [];
```

Comment:
does this need to be a `var` for bridging?
Or could it be made `const`?

```suggestion
const controlValidators: { validator: ControlValidator; ctrlId: string }[] = [];
```